broadcast/internal/outter_errors.go

```diff
@@ -0,0 +1,44 @@
+package outter_errors
+
+import (
+	"fmt"
+	"strings"
+)
+
+func New(msg string, inner error) error {
+	return &outterErr{
+		msg:   msg,
+		inner: inner,
+	}
+}
+
+type outterErr struct {
+	msg   string
+	inner error
+}
+
+func (e *outterErr) Error() string {
+	return e.error(1)
+}
+
+func (e *outterErr) Unwrap() error {
+	return e.inner
+}
+
+func (e *outterErr) error(level uint8) string {
```

Comment:
- Consider rename it to sth like: `formatErrorMessage`. 
- I'm not sure why `uint8` is used here when still, the conversion to "standard" int is made for the `Repeat` func.

Reply:
Changed to `WithCause()`,  please re-review